packages/lesswrong/components/ultraFeed/UltraFeedScoreBreakdown.tsx

```diff
@@ -0,0 +1,172 @@
+import ForumIcon from '../common/ForumIcon';
+import { PostScoreBreakdownContent, ThreadScoreBreakdownContent, scoreBreakdownStyles } from './ScoreBreakdownContent';
+import { useUltraFeedContext } from './UltraFeedContextProvider';
+import AlterBonusesDialog from './AlterBonusesDialog';
+import { useCurrentUser } from '../common/withUser';
+import { userIsAdmin } from '@/lib/vulcan-users/permissions';
+
+const styles = defineStyles('UltraFeedScoreBreakdown', (theme: ThemeType) => ({
+  container: {
+    display: 'inline-flex',
+    alignItems: 'center',
+  },
+  score: {
+    fontSize: 14,
+    fontWeight: 500,
+    color: theme.palette.text.secondary,
+    marginLeft: 8,
+    marginRight: 4,
+  },
+  icon: {
+    cursor: 'pointer',
+    color: theme.palette.grey[400],
+    fontSize: 18,
+    marginLeft: 8,
+    verticalAlign: 'middle',
+    position: 'relative',
+    bottom: 1,
+  },
+  tooltip: {
+    backgroundColor: theme.palette.background.paper,
+    borderRadius: 3,
+    boxShadow: theme.palette.boxShadow.lwCard,
+  },
+  headerText: {
+    fontSize: 16,
+    marginBottom: 12,
+    color: theme.palette.text.primary,
+    fontStyle: 'italic',
+  },
+  constraints: {
+    fontSize: 12,
+    color: theme.palette.primary.main,
+    marginTop: 8,
+    fontStyle: 'italic',
+  },
+}));
+
+const ScoreBreakdownTooltip = ({ 
+  children, 
+  headerText, 
+  constraints, 
+  onAlterBonuses 
+}: { 
+  children: ReactNode;
+  headerText?: string;
+  constraints: string[];
+  onAlterBonuses: () => void;
+}) => {
+  const classes = useStyles(styles);
+  const breakdownClasses = useStyles(scoreBreakdownStyles);
+  
+  return (
+    <div className={breakdownClasses.tooltipContent}>
+      {headerText && (
+        <div className={classes.headerText}>
+          {headerText}
+        </div>
+      )}
+      {children}
+      {constraints.length > 0 && (
+        <div className={classes.constraints}>
+          {headerText ? 'Thread constraints' : 'Constraints'}: {constraints.join(', ')}
+        </div>
+      )}
+      <button 
+        className={breakdownClasses.alterBonusesButton}
+        onClick={(e) => {
+          e.stopPropagation();
+          onAlterBonuses();
+        }}
+      >
+        Alter Bonuses
+      </button>
+    </div>
+  );
+};
+
+const UltraFeedScoreBreakdown = ({ metadata, isFirstCommentInThread, sources, commentMetaInfo, postMetaInfo }: { 
+  metadata: RankedItemMetadata;
+  isFirstCommentInThread?: boolean;
+  sources?: FeedItemSourceType[];
+  commentMetaInfo?: FeedCommentMetaInfo;
+  postMetaInfo?: FeedPostMetaInfo;
+}) => {
+  const classes = useStyles(styles);
+  const breakdownClasses = useStyles(scoreBreakdownStyles);
+  const { showScoreBreakdown, setShowScoreBreakdown } = useUltraFeedContext();
+  const [dialogOpen, setDialogOpen] = useState(false);
+  const currentUser = useCurrentUser();
+  
+  if (!userIsAdmin(currentUser)) {
+    return null;
+  }
+  
+  if (isFirstCommentInThread === false) {
+    return null;
+  }
```

Comment:
If this is the case, why are we even passing it in as a prop instead of not rendering it at the callsite?